src/ansi-c/library/unistd.c

```diff
@@ -191,6 +191,7 @@ inline ssize_t _write(int fildes, const void *buf, size_t nbyte)
 // below
 #ifdef _MSC_VER
 #define ssize_t signed long
+#define size_t unsigned
 #else
```

Comment:
'unsigned' isn't actually size_t in Visual Studio, which is confusing. Furthermore, write() doesn't actually return ssize_t, but 'int' in Visual Studio; this is even the case when using x86, and then despite the fact that 'int' only has 32 bits.

Reply:
I've fixed all the types of read, _read, write, _write in the library to match what the documentation says. Now waiting for CI to pass.